src/Resources/views/Menu/sonata_menu.html.twig

```diff
@@ -1,43 +1,43 @@
 {% extends 'knp_menu.html.twig' %}
 
 {% block root %}
-    {%- set listAttributes = item.childrenAttributes|merge({'class': 'sidebar-menu', 'data-widget': 'tree'}) %}
-    {%- set request        = item.extra('request') ?: app.request %}
+    {%- set listAttributes = item.childrenAttributes|merge({'class': 'sidebar-menu', 'data-widget': 'tree'}) -%}
```

Comment:
I'm not sure of the impact of adding `-` in all the tag of `block root`, `block item`, etc.

Maybe it's better to limit the change to `block label` in order to avoid regression.
But I let the choice to other reviewers.